content/docs/user-guide/scripting/script-canvas/editor-reference/text-replacement.md

```diff
@@ -1,126 +1,133 @@
-                "category": "AWS Game Lift",
-                "name": "Player"
-            },
-            "methods": [
-                ... STRIPPED FOR SIZE ... (See Appendix 1 for complete file before/after)
-            ]
-        }
-    ]
-}
-```
-After saving the changes to `AWSGameLiftPlayer.names`, close the Script Canvas Editor and open it again.
+1. After saving the changes to `AWSGameLiftPlayer.names`, close Script Canvas Editor and open it again.
 
-The nodes are now in the proper category with the given name:
+    The nodes are now in the proper category with the given name:
 
-![Image 3](/images/user-guide/scripting/script-canvas/text-replacement-3.png)
+    ![Image 3](/images/user-guide/scripting/script-canvas/text-replacement-3.png)
 
+### Example: Updating translation files
 
-## Example: Updating translation files
+In this scenario, you don't need to generate a new translation file. Instead you only need to edit it.
 
-In this case, we do not need to generate a new translation file, instead we just need to edit it.
+1. Right-click on any of the nodes under AWSGameLiftPlayer.
 
-![Image 2](/images/user-guide/scripting/script-canvas/text-replacement-1.png)
+    ![Image 2](/images/user-guide/scripting/script-canvas/text-replacement-1.png)
 
-Select _"Explore Translation Data"_ it will open a file explorer on the file that contains the relevant translation data, in this case, *AWSGameLiftPlayer.names*.
+1. Choose **Explore Translation Data** in the context menu. It will open a File Explorer window on the file that contains the relevant translation data. In this example, the file is `AWSGameLiftPlayer.names`.
 
-Open the file in your preferred text editor.
+1. Open the file in your preferred text editor.
 
-Find and update any desired elements. 
+1. Find and update any desired elements.
 
-## Best Practices
+1. Save your changes to `AWSGameLiftPlayer.names`, close Script Canvas Editor, and open it again. You should see your updates to the node(s).
 
-* Do not modify the `base`, `context`, `variant` or `typeid` fields
-* Use `Camel Case` syntax with spaces
-* Add the `tooltip` field to provide details and context
-* Use the `category` field to organize nodes
-* Review your changes by closing and reopening the Script Canvas Editor.
-* Watch for Asset Processor errors on .names files, ensure JSON syntax is correct
-* Commit .names files to source control, this will ensure they are tracked and changes are not overriden
-* Generating translation data will overwrite existing translation data, be careful not to lose work, when in doubt, use _Explore Translation Data_ first
-* Different kinds of nodes will produce different kinds files, some will have one file for one node, others will have many nodes within a single file. 
+## Best practices
 
+Use the following text replacement best practices when working with `.names` files.
 
+* Do not modify the `base`, `context`, `variant`, or `typeid` fields.
+* Use camel case syntax with spaces.
```

Comment:
Would it be better to simplify this to "Capitalize each word in the string" or something to that effect? Many people understand camelCase to mean the first letter of the first word is lower case and subsequent words are capitalized. For them, capitalizing the first letter of every word is PascalCase.